release-scripts/build-conda-tgz.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Script used to install cmdstan on a GColab Machine and create a .tgz archive with the installation
+
+# Arguments:
+# $1: cmdstan version ( will determine archive name )
+
+# On a fresh colab instance simply run:
+# !curl https://raw.githubusercontent.com/stan-dev/ci-scripts/conda-tgz-script/release-scripts/build-conda-tgz.sh | bash -s -- 2.29.0
+# You can follow all the progress in the console, should take around 10 minutes
+# At the end the console will print a URL where you can download the .tgz archive
+
+VERSION="$1"
+INSTALLATION_HOME="/root"
+
+echo "Creating v$VERSION conda .tgz archive !"
+
+# Install cmdstanpy
+pip install --upgrade cmdstanpy
+
+# Install cmdstan
+python <<HEREDOC
+from cmdstanpy import install_cmdstan
+install_cmdstan(cores=2, progress=True)
+HEREDOC
+
```

Comment:
We may want to actually specify the version here?

Also, is there a reason to not use `python -c "from cmdstanpy import install_cmdstan;
install_cmdstan(cores=2, progress=True)"`

Reply:
No other reason than readability, we can change it tho